### ISSUE TITLE
Skywalking for Hashicorp Nomad

### ISSUE THREAD

**Issue by thatsk** [apache/skywalking#9919]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Hi, I would like to Configure hashicorp nomad for container-based deployment.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Hi, thanks for proposing.
We have docker, helm, and k8s operator deployment in separate repository.
I would like to ask, is this going to be another kind of deployment description?

**Comment by thatsk** [comment]
Yes it's driven by nomad jobs and it's another container orchestration which is much much smaller than k8s

**Comment by thatsk** [comment]
https://www.nomadproject.io/ 

If you need any help i would really like to help

**Comment by wu-sheng** [comment]
If it is separated, then we need at least on dev mail list, because we need another repository for it.
Would you like to subscribe to dev mail list and propose this officially?

This is one you could ref 

https://lists.apache.org/list?dev@skywalking.apache.org:lte=3M:Propose%20to%20create%20SkyWalking%20PHP%20agent%20project

Notice, this was proposed by maintainer, so, it was easy to be approved.

Your proposal may need to explain why and whether nomad is widely used, whether this is easy to keep being well maintained like other sub projects.
